packages/mongoose-store/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# How to contribute
```

Comment:
Thanks for catching this, I'll go ahead and try to get all of the possible files up to the top.